src/test/java/net/consensys/orion/config/TomlConfigTest.java

```diff
@@ -136,10 +136,16 @@ void invalidConfigsThrowException() {
     final ConfigException e = assertThrows(
         ConfigException.class,
         () -> Config.load(this.getClass().getClassLoader().getResourceAsStream("invalidConfigTest.toml")));
-    final String message = "Value of property 'clienturl' is not a valid URL (line 4, column 1)\n"
+    final String tuweniPrefix = "org.apache.tuweni.config.ConfigurationError: ";
+    final String message = tuweniPrefix
+        + "Value of property 'clienturl' is not a valid URL (line 4, column 1)\n"
+        + tuweniPrefix
         + "Value of property 'storage' must have storage type of \"leveldb\", \"mapdb\", \"sql\" or \"memory\" (line 11, column 1)\n"
+        + tuweniPrefix
         + "Value of property 'othernodes' is not a valid URL (line 6, column 1)\n"
+        + tuweniPrefix
         + "Value of property 'othernodes' is not a valid URL (line 6, column 1)\n"
+        + tuweniPrefix
```

Comment:
this changed in tuweni - does this look like normal behavior for toml parsing?

Reply:
It chagned in 1.3.  We need to do a similar change in besu.

Reply:
This line here: https://github.com/mark-terry/orion/blob/ca0f6156c1dbb3d3856e2dd28fbf99c98eb188c6/src/main/java/net/consensys/orion/config/Config.java#L81
Change `ConfigurationError::toString` to `ConfigurationError::getMessage` to get the message instead.

Sorry, I had all sorts of errorprone errors after upgrading.

Reply:
Thanks for picking this up. I've made a new PR for this change: https://github.com/ConsenSys/orion/pull/401